src/FortTroopCalculator.tsx

```diff
@@ -0,0 +1,394 @@
+                      setTroops({
+                        ...troops,
+                        totalTroops: parseFloat(e.target.value),
+                      })
+                    }
+                    type="number"
+                    step="25000"
+                    min="0"
+                    max="375000"
+                    name="totalTroops"
+                    id="totalTroops"
+                    placeholder="200000"
+                    value={troops.totalTroops}
+                    className="mt-1 block w-full rounded-md border-gray-300 shadow-sm focus:border-sky-500 focus:ring-sky-500 sm:text-sm"
+                  />
+                </div>
+                <div className="col-span-6 sm:col-span-4">
+                  <label
+                    htmlFor="t5Percent"
+                    className="block text-sm font-medium text-gray-700"
+                  >
+                    T5 Percent
+                  </label>
+                  <input
+                    onChange={(e) =>
+                      setTroops({
+                        ...troops,
+                        t5Troops: parseFloat(e.target.value),
+                      })
+                    }
+                    type="number"
+                    step="10"
+                    min="0"
+                    value={troops.t5Troops}
+                    max="100"
+                    placeholder="60"
+                    name="t5Percent"
+                    id="t5Percent"
+                    className="mt-1 block w-full rounded-md border-gray-300 shadow-sm focus:border-sky-500 focus:ring-sky-500 sm:text-sm"
+                  />
+                </div>
+              </div>
+              <div className="bg-gray-50 px-4 py-3 text-right sm:px-6">
+                {result.t5inf === 0 && (
+                  <span className="text-slate-600 mr-4">
+                    Enter a correct comp to calculate
+                  </span>
+                )}
+
+                <button
+                  onClick={() => {
+                    setTroops(resetTroops);
+                    setResult(resetResult);
+                  }}
+                  className={`inline-flex justify-center rounded-md border border-transparent px-4 mx-2 text-base font-medium text-sky-600 hover:bg-sky-50`}
+                >
+                  Reset
+                </button>
+                <button
+                  disabled={totalComp !== 20 && totalComp !== 10}
+                  onClick={calculateTroops}
+                  className={`disabled:opacity-75 inline-flex justify-center rounded-md border border-transparent bg-sky-600 py-2 px-4 text-sm font-medium text-white shadow-sm hover:bg-sky-700 focus:outline-none focus:ring-2 focus:ring-sky-500 focus:ring-offset-2`}
+                >
+                  Calculate
+                </button>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      {result.t5inf + result.t5range + result.t5cav > 0 && (
```

Comment:
what if they have 0 t5 and only t4?

Reply:
Then the calculator won't be that usefull for someone just with t4 but will fix to apply

Reply:
it will be helpful. imagine filling 375k t4 and you need to fill 659 or 974 or whatever